### PR TITLE
running multiple samples with fio-d

### PR DESCRIPTION
**Pull request by aakarshg**
Also takes care of indexing the results from multiple samples, but doesn't do any statistics.

### CODE REVIEW

**Comment by aakarshg** [comment]
Tested locally, looks good. results in elasticsearch - http://marquez.perf.lab.eng.rdu2.redhat.com:5601/goto/a5f5641147d5cbbc10c030a310884dba

File: roles/fio-distributed/tasks/main.yml [hunk 1]
```diff
@@ -104,12 +104,19 @@
     api_version: batch/v1
     namespace: '{{ operator_namespace }}'
     label_selectors:
-      - app = fiod-client
+      - app = fiod-client-{{fiod_sample}}
```
Comment (dry923) [inline roles/fio-distributed/tasks/main.yml]: Not to nit-pick but consistency should be -{{ fiod_sample }}

File: playbook.yml [hunk 2]
```diff
@@ -50,6 +50,9 @@
 
     - include_role:
         name: "fio-distributed"
+      loop: "{{ range(0, (workload.args.samples | int)) | list | flatten(1)}}"
```
Comment (dustinblack) [inline playbook.yml]: I don't think the loop control belongs in the `playbook.yaml`. We've used a different convention with at least uperf and pgbench, leveraging jinja2 loops in the template. I think the same thing can be applied to fio-d in the client.yaml file.

Also... Wouldn't looping here result in a separate UUID generated for each loop iteration and mess up the indexing?

**Comment by aakarshg** [comment]
holding until we can come to a consensus on what `rerun`/`samples`/`iteration` mean. This PR assumes `samples` mean rebuilding/configuring infra needed to run the benchmark, instead of just running the benchmark.

**Comment by aakarshg** [comment]
This is temporarily put on hold, as this PR doesn't just run fio `n` times but rather recreates the configuration and then runs fio. #137 achieves the `samples` aspect, and if theres any need for the `rerun` then we shall focus on this PR.

**Comment by dustinblack** [comment]
IMO we should close this one.

**Comment by dustinblack** [comment]
@aakarshg Can we close this out? I don't think there is code here that we'll end up using.

**Comment by aakarshg** [comment]
Closing this, given that theres been no ask.

**Pull request closed without merging**